en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/security/api-authentication/api-key-authentication.md

```diff
@@ -0,0 +1,142 @@
+# API Key Authentication
+
+API Key Authentication in WSO2 Choreo Connect is a simple authentication scheme that accepts a valid self-contained JWT token issued for accessing APIs. Since self-contained tokens can have all authorization information, the token processing time is fast and more efficient. Common practice is to have a short expiration time for self-contained access tokens.
+
+Follow the instructions below to use API Key Authentication with WSO2 Choreo Connect:
+
+## Step 1 - Define API Key Authentication for the API
+
+Use one of the following two methods to enable API Key Authentication in Choreo Connect.
+
+### Method 1 - Using OpenAPI definition security schemes
+
+Follow the instructions below to enable API Key Authentication using OpenAPI definition security schemes.
+
+-  **Step 1 - Define the API Key security scheme**
+
+     Security schemes must be defined on the OpenAPI definition under `securitySchemes`. One or more API key security schemes can be used (as in logical OR) simultaneously. Define the API Key security scheme as follows:
+
+      | Parameter | Description |
```

Comment:
@slahirucd7 , confirm whether the wording in the header row in the table is ok.

Reply:
In OAS GitHub page they are using `Field Name` instead of the word `parameter`. I think it's better if we can use the same name in docs also.

Reply:
Fixed https://github.com/wso2/docs-apim/pull/4958/commits/258fd98ad29e1d9e79e4301ec99abb97c1d246d8